packages/terra-image/src/Image.jsx

```diff
@@ -29,6 +29,14 @@ const propTypes = {
    * Sets the width of the image.
    */
   width: PropTypes.string,
+  /**
+   * Function to be executed when the image load is successful.
+   */
+  onLoad: PropTypes.func,
```

Comment:
I don't think they need to be explicitly set to undefined as they will be undefined if they aren't passed.